index.html

```diff
@@ -1,9 +1,11 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Hello World</title>
+    <title>Hello </title>
```

Comment:
Lol, remove the whitespace, make a commit, and push to this branch again. You will see the pull request update itself and I will review again. :dancer: